webapp/app.py

```diff
@@ -14,24 +10,14 @@
 
 
 def create_app(testing=False):
```

Comment:
While you're at it, would you be happy to remove `create_app`? See some discussion here - https://github.com/canonical-web-and-design/docs.snapcraft.io/pull/132#discussion_r271628269.

Or if you want to discuss it further, just leave it for now and we'll merge this without that change.